pkg/kritis/cryptolib/verifier.go

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cryptolib
+
+import "errors"
+
+// Verifier contains methods to validate an Attestation.
+type Verifier interface {
+	// VerifyAttestation verifies whether an Attestation satisfies at least one
+	// of the public keys under an image. This function finds the public key
+	// whose ID matches the attestation's PublicKeyID, and uses this key to
+	// verify the signature.
+	VerifyAttestation(att *Attestation) error
+}
+
+type KeyType int
```

Comment:
If signature algorithm has a dedicated file, then should this also have it?  Same file maybe?

Reply:
Done.